src/test/java/fi/fmi/avi/archiver/message/populator/FixedDurationValidityPeriodPopulatorTest.java

```diff
@@ -0,0 +1,66 @@
+package fi.fmi.avi.archiver.message.populator;
+
+import com.google.common.testing.NullPointerTester;
+import fi.fmi.avi.archiver.file.InputAviationMessage;
+import fi.fmi.avi.archiver.message.ArchiveAviationMessage;
+import fi.fmi.avi.model.MessageType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FixedDurationValidityPeriodPopulatorTest {
+
+    private FixedDurationValidityPeriodPopulator fixedDurationValidityPeriodPopulator;
+    private final InputAviationMessage inputAviationMessage = InputAviationMessage.builder().buildPartial();
+
+    @BeforeEach
+    public void setUp() {
+        fixedDurationValidityPeriodPopulator = new FixedDurationValidityPeriodPopulator(MessagePopulatorTests.TYPE_IDS,
+                MessageType.SPACE_WEATHER_ADVISORY, Duration.ofHours(30));
+    }
+
+    @Test
+    public void swx_validity_end() {
+        final ArchiveAviationMessage.Builder aviationMessage = ArchiveAviationMessage.builder()
+                .setMessageTime(Instant.parse("2019-05-10T00:00:00Z"))
+                .setType(MessagePopulatorTests.TYPE_IDS.get(MessageType.SPACE_WEATHER_ADVISORY));
```

Comment:
Using `MessagePopulatorTests.TYPE_IDS` map, I'd recommend using `MessagePopulatorTests.TypeId` instances for `MessageType` and id number values throughout the test class to ensure test correctness and better readability.

e.g. here

```suggestion
                .setType(MessagePopulatorTests.TypeId.SWX.getId());
```

Reply:
Fixed 👍